util/events/libuv/FileNo_admin.c

```diff
@@ -49,6 +55,18 @@ static void onResponse(struct FileNo_Promise* promise, int tunfd)
     struct FileNoRequest* fr= Identity_check((struct FileNoRequest*)promise->userData);
     Dict* resp = Dict_new(promise->alloc);
 
+    String* path = String_printf(fr->ctx->alloc, "/proc/%d/fd", getpid());
+    DIR* dir = opendir(path->bytes);
+    if (!dir) {
+        Log_warn(fr->ctx->logger, "Unable to opendir() [%s] [%s]", path->bytes, strerror(errno));
+    } else {
+        struct dirent* ent;
+        while ((ent = readdir(dir))) {
+            Log_debug(fr->ctx->logger, "[type=%d] %s/%s", ent->d_type, path->bytes, ent->d_name);
+        }
+        closedir(dir);
+    }
```

Comment:
Same comment as above, also you know that normally cjdns is chrooted into an empty directory, right ?